pkg/minishift/util/b2d.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/docker/machine/libmachine/drivers"
+)
+
+func IsB2d(driver drivers.Driver) bool {
```

Comment:
I'd rather see a check which takes the values of /etc/os-release. As comparing can be done in code here.

Reply:
IsB2D, pretty much stylized as IsCentOS would also be if implemented.